provider/cmd/pulumi-resource-aws/schema.json

```diff
@@ -261784,7 +261784,7 @@
                     },
                     "name": {
                         "type": "string",
-                        "willReplaceOnChanges": true
+                        "deprecationMessage": "This property has been deprecated. Please use 'dbName' instead."
```

Comment:
I think as written it will still replace on changes, no? Because you copy the value into dbName which does replace on changes. Can we get "willREplaceOnChanges" marker back? I know it's only for docs.

Reply:
It will. I'll add that message back in.